lib/internal/errors.js

```diff
@@ -129,6 +129,8 @@ E('ERR_INVALID_OPT_VALUE',
   (name, value) => {
     return `The value "${String(value)}" is invalid for option "${name}"`;
   });
+E('ERR_INVALID_REPL_EVAL_CONFIG',
+  'Cannot specify both breakEvalOnSigint and eval for REPL');
```

Comment:
I was using the original text of the error, but I agree that it should use the quotation marks (I wasn't aware of the convention). Thanks!